### PR TITLE
Revert Dynamic Validators Configuration

### PR DESCRIPTION
**Pull request by kjezek**
This PR reverts configuration of validatros. 

Current version is not working properly, it creates validators twice, when only one is actlually configured in a scenaario file.  It leads to adhoc failures, where blocks are not produced. Althought it was not fully investigated, the problem seems to be that one of the validators is not properly configured and it brakes the network.

Furthermore, the code is not tested.

Originally, it was possible to configure only a fixed  number of validators that initialised the network and run for the whole scenario.  

Since this was working, this PR reverts to this original configuration.

### CODE REVIEW

File: driver/executor/executor.go [hunk 1]
```diff
@@ -307,74 +307,6 @@ func scheduleNodeEvents(node *parser.Node, queue *eventQueue, net driver.Network
 
 		queue.add(nodeCreate)
 
-		// 2. Queue Timer SimEvents
```
Comment (rpl-ffl) [inline driver/executor/executor.go:310]: This removes the timer config, which is separate from validator config.

I understand the rationale for this change, but it would make sense to make a different PR and remove any usage of timer from any scenarios there.

**Review by rpl-ffl — CHANGES_REQUESTED** [review]

File: driver/executor/executor.go (hunk 1 above)
Comment (kjezek) [reply driver/executor/executor.go:310]: yes, I can put it in another PR

Comment (rpl-ffl) [reply driver/executor/executor.go:310]: I can handle the the timer config revert PR.

Comment (kjezek) [reply driver/executor/executor.go:310]: ok, thanks

File: scenarios/small_new_validator.yml
Comment (HerbertJordan) [inline scenarios/small_new_validator.yml:1]: This scenario should be removed since it is not doing what its name suggests.

File: driver/parser/check.go [hunk 2]
```diff
@@ -466,29 +424,10 @@ func checkTimeInterval(start, end *float32, duration float32) error {
...
-	// count static validators within the node
-	gvCount, dynamicValidatorCount := s.GetStaticDynamicValidatorCount()
-	if s.NumValidators == nil {
-		s.NumValidators = &gvCount
-	}
-
-	// This must be check here for test to pass
-	// _must_ allow case where validator count = 0
-	if *s.NumValidators < 0 {
+	if s.NumValidators != nil && *s.NumValidators < 0 {
```
Comment (HerbertJordan) [inline driver/parser/check.go:428]:
```suggestion
	if s.NumValidators != nil && *s.NumValidators < 1 {
```

Also, the error message needs to be updated.

**Review by HerbertJordan — APPROVED** [review]
Looks good, thanks!

File: scenarios/small_new_validator.yml
Comment (kjezek) [reply scenarios/small_new_validator.yml:1]: right

**Review by rpl-ffl — APPROVED** [review]
Thank you!
